src/algebra/order/lattice_group.lean

```diff
@@ -3,7 +3,7 @@ Copyright (c) 2021 Christopher Hoskin. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Christopher Hoskin
 -/
-import algebra.group_power.basic -- Needed for squares
```

Comment:
Is this intended?

Reply:
No, the algorithm didn't attempt to preserve comments when it had to rewrite an import block. I'll fix this now.